.tasks

```diff
@@ -13,8 +13,8 @@
 
 set -ex
 
-# Scan for all tests
-bots/tests-scan
+# Process exactly one item from the queue
+bots/tests-dequeue-one --amqp cockpitrabbitmq.cockpit.svc.cluster.local --queue tests
```

Comment:
bikeshed alert.. As the main purpose of this is to *run* a test (or other action), and dequeuing is the means, plus in the future this should do "issue-scan" like things, too -- how about naming this "bots/run-queue"?

I'm not sure if we should have separate queues for separate kinds of tasks (e. g "po-refresh", "image-refresh", "test"), or have a queue item JSON contain a type field, or just contain a completely opaque shell script. My gut feeling is that the latter is easiest for now, and have at most two queues (tasks and redhat-tasks). We can refine the queue structure later on, if that at some point becomes impractical. WDYT?